aesara/breakpoint.py

```diff
@@ -35,9 +35,9 @@ class PdbBreakpoint(Op):
 
     .. code-block:: python
 
-        import theano
-        import theano.tensor as tt
-        from theano.breakpoint import PdbBreakpoint
+        import aesara
+        import aesara.tensor as tt
```

Comment:
do we want to update these imports in docs to the proposal in #148 `import aesara as ae; import aesara.tensor as aet`?

Reply:
Yeah, I was thinking of `at`, though, but `ae*` isn't bad.

Reply:
It does not need to be decided here nor now, but probably whatever is used in the docstrings (herr and at pymc3) will be what ends up sticking around